rp2040-hal/src/gpio/pin/pin_sealed.rs

```diff
@@ -43,8 +43,8 @@ macro_rules! accessor_fns {
                 unsafe {
                     let sio = &*$crate::pac::SIO::PTR;
                     match pin.bank {
-                        DynBankId::Bank0 => &sio.[<gpio_ $reg:lower>](),
-                        DynBankId::Qspi => core::mem::transmute(&sio.[<gpio_hi_ $reg:lower>]()),
+                        DynBankId::Bank0 =>sio.[<gpio_ $reg:lower>](),
```

Comment:
It looks like cargo fmt ignores these macros. Otherwise it would have added line-breaks to the very long transmute expressions.
(I considered adding line-breaks manually, but IMHO that wouldn't improve readability. This blend of generic parameters and paste expressions looks like line noise either way.)